pkgs/development/libraries/hunspell/dictionaries.nix

```
@@ -967,4 +967,31 @@ rec {
       platforms = platforms.all;
     };
   };
+
+    /*
+  Turkish
+  */
+  tr_TR = tr_tr;
+  tr_tr = mkDict rec {
```

```suggestion
  tr-tr = mkDict rec {
```

Other packages write this way?

---

Some of the other packages where written this way. I mainly based it on the NL language support. But it should not matter, your version should work too.